src/main/scala/com.snowplowanalytics.sauna/responders/urbanairship/UAResponder.scala

```diff
@@ -0,0 +1,118 @@
+package com.snowplowanalytics.sauna
+package responders
+package urbanairship
+
+import scala.io.Source.fromInputStream
+import java.io.DataOutputStream;
+import org.apache.commons.io.IOUtils;
+import java.io.File;
+import java.util.Properties;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.io.InputStream;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.net.URL;
+import java.util.Map.Entry;
+import scala.io.Source
+import scala.collection;
+import scala.collection.mutable.{ Map => MutableMap }
+import javax.net.ssl.HttpsURLConnection;
+import java.util.zip.GZIPOutputStream
+import apis.UrbanAirship
+import loggers.Logger.Notification
+import akka.actor.{ActorRef, Props}
+import responders.Responder.FileAppeared
+import responders.urbanairship.UAResponder.Airship
+import scala.concurrent.ExecutionContext.Implicits.global
+
+
+class UAResponder(urbanairship: UrbanAirship)
+                (implicit logger: ActorRef) extends Responder {
+
+  import UAResponder._
+  
+ val pathPattern =
+    """.*com\.urbanairship/
+      |static_lists/
+      |v1/
+      |tsv:\*/
+      |.+$
+    """.stripMargin
+       .replaceAll("[\n ]", "")
+       
+  
+ override def process(fileAppeared: FileAppeared): Unit = {
+    import fileAppeared._
+   convertTSV(is)
+  }
+   
+  def convertTSV(is: InputStream):Unit = {
+      
+    
+    var appToListMap = Map[String, Map[String, List[Airship]]]()
+
+    var listToIdentifierMap = Map[String, List[Airship]]()
+    
+    var count=0
+
+    for (line <- fromInputStream(is).getLines()) {
+
+      val rawlines = line.split("\t",-1)
+
+      val lines = for (i <- rawlines) yield i.substring(1, i.length() - 1)
```

Comment:
If you want to trim double quotes here, you probably should use `line.trim('"')`. It provides clues to maintainer what are trying to achieve here and also safer because you don't accidentally trim something important.